src/raftstore/store/worker/apply.rs

```diff
@@ -753,6 +780,21 @@ impl ApplyDelegate {
                     self.metrics.size_diff_hint = 0;
                     self.metrics.delete_keys_hint = 0;
                 }
+                ExecResult::PrepareMerge { ref region, .. } => {
+                    self.region = region.clone();
+                    self.is_merging = true;
+                }
+                ExecResult::CommitMerge {
+                    ref region,
+                    ref source,
+                } => {
+                    self.region = region.clone();
+                    ctx.merged_regions.push(source.get_id());
```

Comment:
need to clear is_merging here?

Reply:
The peer is going to be destroyed, it doesn't matter.